File: JobStats.WebClient/src/app/types/dateDay.ts
export interface DateDay {
	day: number,
	month: number,
	year: number
}

export function DateDayToDate(dateDay: DateDay): Date{
	return new Date(dateDay.year,dateDay.month,dateDay.day);
}

export function DateToDayDate(date:Date): DateDay {
	return {year: date.getFullYear(), month: date.getMonth(), day: date.getDate()};
}

File: JobsStats.API/JobsStats.API/Types/DateDay.cs
namespace JobsStats.API.Types
{
    public class DateDay
    {
        public int Day { get; set; }
        public int Month { get; set; }
        public int Year { get; set; }

        public static DateDay DateTimeToDateDay(DateTime dateTime) =>
    new DateDay { Day = dateTime.Day, Month = dateTime.Month, Year = dateTime.Year };
    }
}
